composer.json

```diff
@@ -15,6 +15,8 @@
         "php-http/httplug": "^2.0",
         "php-http/message-factory": "^1.0",
         "php-http/message": "^1.6",
+        "psr/http-factory": "^1.0",
```

Comment:
strictly speaking, a psr factory implementation will require the psr package as well, and when none is available it does not hurt to not have the package installed. but this being a psr with just a few interfaces, i agree to include it for semantic clarity.